src/entities/Place/routes/getPlaceList.ts

```diff
@@ -86,6 +87,6 @@ export const getPlaceList = Router.memo(
       sceneStats
     )
 
-    return new ApiResponse(response, { total })
+    return new ApiResponse(response, { total: Number(total) })
```

Comment:
we should instead check why `total` is not a number

Reply:
it's not been a number since first place's release. query `count(*)` is returned as a string